yarn.lock

```diff
@@ -5052,18 +5052,17 @@
   resolved "https://registry.yarnpkg.com/@uniswap/token-lists/-/token-lists-1.0.0-beta.21.tgz#5a4844236e328d3b0e87161c19edaace669e3e6e"
   integrity sha512-IQ1YpWKgXIsfYqym9Nl/uaB1t5Wk70pyqJ2W4FmvQmt3FIQz+pHlzlEl91aYloPrlcZaFluE0Jq7gkfYpIjsBw==
 
-"@unstoppabledomains/resolution@3.0.0":
-  version "3.0.0"
-  resolved "https://registry.yarnpkg.com/@unstoppabledomains/resolution/-/resolution-3.0.0.tgz#148080757f3ab2e94993228a5934ea60f1e0807b"
-  integrity sha512-iAvVkDWsSdi9WOiCp8t6GwWQZYIQjca+JAzfnNB7LvRDim3jrgcLlbeSYu+Mwzn2JKUGf/Ygw7YtnmjihnT5fw==
+"@unstoppabledomains/resolution@^7.1.4":
```

Comment:
If I run `yarn` from the repo root, I get a diff in the lockfile—could you commit the updated version?

Reply:
I committed the updated lockfile!